frontend/packages/console-app/src/components/quick-starts/QuickStartDrawer.tsx

```diff
@@ -65,47 +50,22 @@ const QuickStartDrawer: React.FC<QuickStartDrawerProps> = ({
 
   const onModalCancel = () => setModalOpen(false);
 
-  const headerClasses = classNames('co-quick-start-drawer-head', {
-    'pf-u-box-shadow-sm-bottom':
-      scrollDirection && scrollDirection !== ScrollDirection.scrolledToTop,
-  });
-
-  const panelContent = quickStart ? (
-    <DrawerPanelContent onScroll={handleScrollCallback}>
-      <div className={headerClasses}>
-        <DrawerHead>
-          <div className="co-quick-start-drawer__title">
-            <Title
-              headingLevel="h1"
-              size="xl"
-              style={{ marginRight: 'var(--pf-global--spacer--md)' }}
-            >
-              {quickStart?.spec.displayName}{' '}
-              <small className="co-quick-start-drawer__duration text-secondary">
-                {t('quickstart~{{duration, number}} minutes', {
-                  duration: quickStart?.spec.duration,
-                })}
-              </small>
-            </Title>
-          </div>
-          <DrawerActions>
-            <DrawerCloseButton onClick={handleClose} />
-          </DrawerActions>
-        </DrawerHead>
-      </div>
-      <DrawerPanelBody>
-        <AsyncComponent
-          loader={() => import('./QuickStartController').then((c) => c.default)}
-          quickStart={quickStart}
-        />
-      </DrawerPanelBody>
-    </DrawerPanelContent>
-  ) : null;
-
   return (
     <>
       <Drawer isExpanded={!!activeQuickStartID} isInline>
-        <DrawerContent panelContent={panelContent}>
+        <DrawerContent
+          panelContent={
+            <QuickStartsLoader>
+              {(quickStarts) => (
+                <QuickStartPanelContent
+                  quickStarts={quickStarts}
+                  handleClose={handleClose}
+                  activeQuickStartID={activeQuickStartID}
+                />
+              )}
```

Comment:
Move this into panelContent variable above.

Reply:
updated